### ISSUE TITLE
windows 11

### ISSUE THREAD

**Issue by michiel2002v** [dsafa/audio-band#292]
Is it possible Audioband does not work with windows 11?
I just tried to install it, but it does not seem to work...

**Comment by AlizerUncaged** [comment]
yep doesnt work in windows 11, microsoft did a complete rewrite of the taskbar

**Comment by Yuzikus** [comment]
works for me with Start all Back

**Comment by stephenshutters** [comment]
> works for me with Start all Back

What does this mean? What is "Start all Back"? I am trying to get AudioBand to work on Windows 11 and not having any success.

**Comment by stephenshutters** [comment]
> > works for me with Start all Back
> 
> What does this mean? What is "Start all Back"? I am trying to get AudioBand to work on Windows 11 and not having any success.

Never mind. I just found [the answer](https://www.startallback.com/) to my own question.

**Comment by Yuzikus** [comment]
> > > works for me with Start all Back
> > 
> > 
> > What does this mean? What is "Start all Back"? I am trying to get AudioBand to work on Windows 11 and not having any success.
> 
> Never mind. I just found [the answer](https://www.startallback.com/) to my own question.

Also use this version https://github.com/dsafa/audio-band/issues/285

**Comment by windowskater** [comment]
Any info on how this would work now? I tried using StartAllBack for this but I don't find an option to enable AudioBand. Neither in the taskbar (like I would when using ExplorerPatcher with Win10 Taskbar) nor in StartAllBack.
